lk2nd/device/dts/msm8916/msm8916-wiko-chuppito.dts

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: GPL-2.0-only
+ 
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8916 0>;
+	qcom,board-id = <0x1000b 0x05 0x1010b 0x05 0x3010b 0x05>;
```

Comment:
this is 3 different ids, could you please check which of those you actually need and use QCOM_BOARD_ID() macro for the first cell